src/components/MeasurementsToolList.vue

```diff
@@ -4,9 +4,12 @@ import { computed } from 'vue';
 import { useCurrentImage } from '@/src/composables/useCurrentImage';
 import { AnnotationToolStore } from '@/src/store/tools/useAnnotationTool';
 import { frameOfReferenceToImageSliceAndAxis } from '@/src/utils/frameOfReference';
+import useHistoryStore from '@/src/store/history';
+import { createRemoveToolOperation } from '@/src/store/operations/tools';
+import { Store } from 'pinia';
 
 const props = defineProps<{
-  toolStore: AnnotationToolStore<ToolID>;
+  toolStore: AnnotationToolStore<ToolID> & Store;
```

Comment:
In the future, should I put `& Store` on the `AnnotationToolStore` shared type?

Reply:
Yeah, I think that is a good idea. I could add that as an extra commit to this PR if you'd like.